tree/ntuple/v7/test/rfield_vector.cxx

```diff
@@ -1,51 +1,99 @@
 #include "ntuple_test.hxx"
 
-TEST(RNTuple, ClassVector)
+// A layer of indirection to hide std::vector's second template parameter.
+// This way we can generalize tests over RVec and std::vector using a template template parameter (see below).
+template <typename T>
+using Vector_t = std::vector<T>;
+
+template <template <typename> class Coll_t>
+void TestClassVector()
 {
    FileRaii fileGuard("test_ntuple_classvector.root");
```

Comment:
Should use different file names for the different invocations.

Reply:
done!